lib/scala/launcher/src/main/scala/org/enso/launcher/Launcher.scala

```diff
@@ -0,0 +1,68 @@
+package org.enso.launcher
+
+import buildinfo.Info
+import java.nio.file.Path
+
+import org.enso.pkg.PackageManager
+
+object Launcher {
+  private val packageManager = PackageManager.Default
+
+  private def workingDirectory: Path = Path.of(".")
+
+  def newProject(name: String, path: Option[Path]): Unit = {
+    // TODO [RW] this is not the final implementation
+    val actualPath = path.getOrElse(workingDirectory.resolve(name))
+    packageManager.create(actualPath.toFile, name)
+    println(s"Project created in $actualPath")
+  }
+
+  def displayVersion(useJson: Boolean): Unit = {
+    val version   = Info.ensoVersion // Note [Launcher Version]
+    val osArch    = System.getProperty("os.arch")
+    val osName    = System.getProperty("os.name")
+    val osVersion = System.getProperty("os.version")
+
+    val runtimeVersion = // TODO [RW] add with #976
+      if (useJson) "\"<not implemented yet>\""
+      else "Runtime component is not yet implemented in the launcher."
+
+    val versionOutput =
```

Comment:
This is copy paste from runner. Probably worth extracting?

Reply:
Should I create a separate library for it?

Reply:
Well now that I look deeper into it, yes it was copied, but substantial parts of this do differ - we skip the runtime VM name as the launcher is a native binary (well it would probably print 'Substrate VM' or something, but would that be useful as it can be directly inferred from the GraalVM version that it was built with) and it has to actually run `enso.jar --version` too to display the current Enso version (however this part is currently unimplemented, as it will come in a future PR).
We also print the GraalVM version differently, as in the launcher we indicate that it is the Native Image component that is used.

So the only shared part is getting 3 system properties and printing a few lines or JSON - but even there there are important differences, as the Launcher's version string / JSON will in the future contain that of the engine.

Do you think it is worth it to create a separate library just for these 2.5 lines?
```
Built with: scala-${Info.scalacVersion} [and GraalVM ${Info.graalVersion} Native Image / for GraalVM ${Info.graalVersion}] # this one still differs
Built from: ${Info.branch}$dirtyStr @ ${Info.commit}
Running on: $osName $osVersion ($osArch)
```

Of course it can be done if deemed necessary, but I'm not sure if this would be helpful?

Reply:
Yup. Extract a new microlibrary able to print the version string. Make the contents of that string controllable with arguments. This also allows you to cleanup sbt config, because you'll only need one auto-generated file.

Reply:
And since you told me the project-manager also copy-pastes it, it's a great occasion to fix that one as well <3